controllers/httproute_controller.go

```diff
@@ -379,3 +409,45 @@ func isExternalDNSSupported(mgr ctrl.Manager) (bool, error) {
 	}
 	return false, nil
 }
+
+func (r *HTTPRouteReconciler) areBackendRefsIpFamiliesValid(ctx context.Context, httpRoute *gateway_api.HTTPRoute) bool {
+	rules := httpRoute.Spec.Rules
+
+	for _, rule := range rules {
+		backendRefs := rule.BackendRefs
+
+		for _, backendRef := range backendRefs {
+			// For now we skip checking service import
+			if *backendRef.Kind == "ServiceImport" {
+				continue
+			}
+
+			svc := &corev1.Service{}
+
+			glog.Infof("backendRef: %v", backendRef)
+
+			key := types.NamespacedName{
+				// Namespace: string(*backendRef.Namespace),
```

Comment:
why commented?

Reply:
Thanks, I forgot to remove that while I was doing manual testing\